xds/src/main/java/io/grpc/xds/client/XdsClient.java

```diff
@@ -427,6 +431,25 @@ public interface ResourceStore {
     Collection<String> getSubscribedResources(ServerInfo serverInfo,
```

Comment:
Remove this method and replace with the one below with authority in its parameter.